lib/inferno/db/migrations/010_add_validator_sessions.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  change do
+    create_table :validator_sessions do
+      column :id, String, primary_key: true, null: false, size: 36
+      column :validator_session_id,  String, null: false, size: 255, unique: true
+      column :test_suite_id, String, null: false, size: 255
+      column :validator_name, String, null: false, size: 255
+      column :suite_options, String, text: true, size: 255
```

Comment:
Do you think we are likely to encounter issues with the length of this field since it's serialized JSON?

Reply:
Valid concern given the length of options in g10. I will remove the size constraint.